service/controller/release.go

```diff
@@ -53,6 +61,9 @@ func NewRelease(config ReleaseConfig) (*Release, error) {
 			Logger:  config.Logger,
 			Watcher: config.G8sClient.Release().ReleaseCycles(""),
 
+			ListOptions: metav1.ListOptions{
+				LabelSelector: fmt.Sprintf("%s=%s", releaseOperatorVersionLabel, project.Version()),
```

Comment:
We need to support `service.controller.watch.version` setting here.

Reply:
Removed the label selector from this PR, as we spoke and it's more complex than this.